participants/anthony.json

```
@@ -0,0 +1,36 @@
+{
+    // required
+    "name": "Anthony Craig",
+    // optional
+    "company": "your company name",
+    // required
+    "when": {
+        "friday": true,
+        "saturday": false
+    },
+    // required
+    "iCanTakeNotesDuringSessions": false,
+    // required - at least one
+    "tags": ["ReScript"],
+    // optional
+    "vegan": true/false,
+    // optional
+    "vegetarian": true/false,
```

```suggestion
    // optional
    "vegan": false,
    // optional
    "vegetarian": true,
```

You have to select a boolean value here